magellan-library/src/main/java/com/wealthfront/magellan/lifecycle/ActivityLifecycleAdapter.kt

```diff
@@ -0,0 +1,52 @@
+package com.wealthfront.magellan.lifecycle
+
+import android.app.Activity
+import android.widget.FrameLayout
+import androidx.activity.ComponentActivity
+import androidx.annotation.IdRes
+import androidx.lifecycle.DefaultLifecycleObserver
+import com.wealthfront.magellan.core.Navigable
+import androidx.lifecycle.LifecycleOwner
+import com.wealthfront.magellan.ScreenContainer
+
+internal class ActivityLifecycleAdapter(
+  private val navigable: Navigable,
+  private val context: Activity,
+  private val containerRes: Int
+) : DefaultLifecycleObserver {
+
+  override fun onCreate(owner: LifecycleOwner) {
+    navigable.create(context)
```

Comment:
Won't this call onCreate more times than we want it to? (e.g. every time the activity is recreated)